cli/waiter/token_post.py

```diff
@@ -93,13 +95,22 @@ def create_or_update_token(clusters, args, _, enforce_cluster, action):
     token_fields_from_args = args
     shared_keys = set(token_fields_from_json).intersection(token_fields_from_args)
     if shared_keys:
-        raise Exception(f'You cannot specify the same parameter in both an input file '
-                        f'and token field flags at the same time ({", ".join(shared_keys)}).')
+        if not allow_override:
+            raise Exception(f'You cannot specify the same parameter in both an input file '
+                            f'and token field flags at the same time ({", ".join(shared_keys)}) '
+                            f'without specifying the --override flag.')
+        else:
+            logging.debug(f'Following parameters have specified values in both file and flags: {shared_keys}')
 
     token_fields = {**token_fields_from_json, **token_fields_from_args}
     token_name_from_json = token_fields.pop('token', None)
     if token_name_from_args and token_name_from_json:
-        raise Exception('You cannot specify the token name both as an argument and in the input file.')
+        if not allow_override:
```

Comment:
Can there be a test for overriding the token name behavior?

Reply:
Will add.

Reply:
Added test, ready for another round of review.